app/javascript/components/App.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+// import { BrowserRouter as Router, Route } from 'react-router-dom';
```

Comment:
- [ ] Kindly do not push commented out code to your code base, it make your project looks messy and unclean. Therefore I suggest you remove all the commented out code from your project.

Reply:
done